ffig/FFIG.py

```diff
@@ -66,15 +66,16 @@ def get_class_name(header_path):
     return re.sub(".h$", "", header_name)
 
 
-def write_bindings_to_disk(module_name, api_classes, env, output_dir):
+def write_bindings_to_disk(args, api_classes, env, output_dir):
     """
     Write the bindings to disk, return Nothing
     Input:
+    - args from command line
     - api_classes
     - environment to get templates from
-    - args
     - output_dir where to write to
     """
+    module_name = args.module_name
     for binding in args.bindings:
```

Comment:
This looks fine now - apart from the docstring. Go ahead and merge when that has been fixed up.